lib/joint/DistanceJoint.js

```diff
@@ -142,14 +142,28 @@ function DistanceJoint(def, bodyA, bodyB, anchorA, anchorB) {
  */
 DistanceJoint.prototype.getLocalAnchorA = function() {
   return this.m_localAnchorA;
-}
+};
+
+/**
+ * Sets the local anchor point relative to bodyA's origin.
+ */
+DistanceJoint.prototype.setLocalAnchorA = function(anchor) {
+  return this.m_localAnchorA = anchor;
```

Comment:
Is assignment, not copying safe here?

Reply:
getLocalAnchorA returns initial vector, not a clone, so in the same style did assignment here. Used it safely on engine level.